chapters/arrays.tex

```diff
@@ -634,28 +635,25 @@ \subsubsection{Array Constructor with Several Iterators}\doublelabel{array-const
 
 \subsection{Array Concatenation}\doublelabel{array-concatenation}
 
-
-The function \lstinline!cat(k,A,B,C,...)! concatenates arrays
-\lstinline!A!,\lstinline!B!,\lstinline!C!,... along
-dimension k according to the following rules:
+The function \lstinline[mathescape=true]!cat($k$, A, B, C, $\ldots$)! concatenates arrays
```

Comment:
Similarly here - I don't see why k must be as math, and different from A, B, C.

Reply:
(Let's have the discussion under the change above.)

Reply:
There has been a comparatively large number of changes affecting 10.4.2 _Array Concatenation_, but I find it consistent with the rest of the chapter.